app/models/sensor.rb

```diff
@@ -1,4 +1,7 @@
 class Sensor < ApplicationRecord
+  has_many :measurements, dependent: :destroy
```

Comment:
This is the `ON DELETE CASCADE` setting that fixes #46, right?

Reply:
Yes.